04 Strategy Library/00 Strategy Library/01 Strategy Library.php

```diff
@@ -398,6 +398,14 @@
         ],
         'description' => "Invests in equity market 2 days preceding holiday days and stays in cash during the other trading days."
     ],
+    [
+        'name' => 'Term Structure of VIX Futures',
+        'link' => 'strategy-library/term-structure-of-vix-futures',
```

Comment:
This needs to be "Exploiting Term Structure of VIX Futures" --- missing "exploiting"